01_standalone_examples/dagster-integration/docker-compose.yml

```diff
@@ -1,4 +1,5 @@
 version: '3.8'
+name: lakefs-dagster-example
```

Comment:
dup of https://github.com/treeverse/lakeFS-samples/pull/103/files#diff-85cd76686d07243c52681f84edaf54f8ebbdf0cdf4b6187269103f915462a135R20 - should we merge this into #103 ?